find_attitude/find_attitude.py

```diff
@@ -162,7 +250,7 @@ def get_dists_yag_zag(yags, zags, mags=None):
     dists = np.sqrt((yags[idx0] - yags[idx1]) ** 2 + (zags[idx0] - zags[idx1]) ** 2)
     dists = np.array(dists)
     if mags is None:
-        mags = np.ones_like(dists) * 15
+        mags = np.ones_like(yags) * 15
```

Comment:
Is 15 really a default MAG that should be set with a config somewhere?

Reply:
What the "corollary info" is and what value it has is a little confusing here.

Reply:
The 15 is just an arbitrary value for the use case of this being called without the mags. An alternative would be to not provide the `mag0` and `mag1` values in this case, but it makes no difference anyway.

Reply:
The "corollary info" is the indexes and mags.